src/macfc-security-hub-sync.ts

```diff
@@ -446,7 +446,7 @@ export class SecurityHubJiraSync {
 
     // Validate the extracted region
     if (!isAwsRegion(region)) {
-      console.error(`Invalid AWS region: ${region}`)
+      //console.error(`Invalid AWS region: ${region}`)
```

Comment:
Any reason why this line needs to be commented out?

Reply:
The logs looked so ugly and were used to indicate the finding url type where the finding id does not contain the region. However, our logic sets the region to us-east-1 as fall-back logic so to keep the logs in run as short as possible I removed this extra log
![image](https://github.com/user-attachments/assets/7848e629-e8ec-47ed-a97e-abcb36d66a4e)

Reply:
cleaner logs helps w debugging when needed